libraries/eosiolib/contract.hpp

```diff
@@ -35,7 +35,7 @@ class contract {
        * @param n - The name of this contract
        * @param ds - The datastream used 
        */
-      contract( name n, datastream<const char*> ds ):_self(n), _ds(ds) {}
+      contract( name n, name code, datastream<const char*> ds ):_self(n), _ds(ds) {}
```

Comment:
Missing initialization: `_code(code)`. Similar change as above regarding doxygen comment and renaming `n`.